src/controllers/sports.ts

```diff
@@ -72,14 +72,14 @@ async function updateSport(req: Request, res: Response) {
 }
 
 /**
- * Eliminar un elemento de la base de datos
+ * Delete an sport item from the database
  * @param req
  * @param res
  */
 async function deleteSport(req: Request, res: Response) {
   try {
     await models.sports.findOneAndDelete({ _id: req.params.id });
-    res.send({ message: 'DELETED_SUCCESFULLY' });
+    res.send({ message: 'Sport deleted successfully' });
```

Comment:
If you think it is necessary to do some tweaks on the code, create a new ticket fixing all those issues, instead of mixing all the improvements with the feature development

Reply:
Understood, if I want to make changes to the issues to facilitate friendlier communication between users, I will create the corresponding ticket if necessary.